tests/test_map.py

```diff
@@ -114,3 +115,37 @@ def flatten(tree: NodeTreeDto) -> List[str]:
 
     result = flatten(await api.maps.get_map_nodes(m.id, root_id=n2.id))
     assert result == [n2.id, n3.id]
+
+
+@pytest.mark.asyncio
+async def test_search_nodes(secret: Secret, api: RfApiClient):
+    m = await prepare_map(api, secret.developer_prefix, 'test_search_nodes')
+
+    p = CreateNodePropertiesDto.empty()
+    p.global_.title = 'first node'
+    await api.nodes.create(CreateNodeDto(
+        map_id=m.id,
+        parent=m.root_node_id,
+        type_id=None,
+        position=(PositionType.R, '1'),
+        properties=p
+    ))
+
+    p = CreateNodePropertiesDto.empty()
+    p.global_.title = 'second node'
+    await api.nodes.create(CreateNodeDto(
+        map_id=m.id,
+        parent=m.root_node_id,
+        type_id=None,
+        position=(PositionType.R, '1'),
+        properties=p
+    ))
+
+    # для того что бы рф успел проиндексировать узлы
```

Comment:
All comments should be written in English. In an international environment, English is the preferred language